src/StmtToHtml.cpp

```diff
@@ -142,9 +142,14 @@ class StmtToHtml : public IRVisitor {
         stream << op->value;
         stream << close_span();
     }
+    void visit(const UIntImm *op){
+        stream << open_span("UIntImm Imm");
+        stream << op->value;
+        stream << close_span();
+    }
     void visit(const FloatImm *op){
         stream << open_span("FloatImm Imm");
-        stream << op->value << 'f';
+        stream << op->value;
```

Comment:
Do we want to gate whether the 'f' appears based on the bits in the immediate? (And float16_t too perhaps?)